src/_pytest/_io/terminalwriter.py

```diff
@@ -223,7 +223,7 @@ def _highlight(
                         style=os.getenv("PYTEST_THEME"),
                     ),
                 )
-                return highlighted
+                return "\x1b[0m" + highlighted
```

Comment:
Let's add a comment here regarding the reset (can use what you wrote in the commit message).

Reply:
Added something, how does that sound?